src/interface/substrate_query.js

```diff
@@ -395,6 +409,12 @@ module.exports = {
                 } catch (e) {
                     return {'error': e.toString()};
                 }
+	    case 'staking/bonded':
```

Comment:
Since in the case `param2` is a required parameter, we need to check whether the user entered it or not. I suggest the following implementation:

```
	    case 'staking/bonded':
                try {
                    if (!param2) {
                        return {'error': 'You did not enter the stash '
                                + 'address that needs to be queried'};
                    }
                    return {'result': await getStakingBonded(api, param2)};
                } catch (e) {
                    return {'error': e.toString()};
                }
```

Reply:
Would appreciate if you could test this out.